lib/api/api.dart

```diff
@@ -331,6 +332,9 @@ class ApiSingleton {
       if (report.selected_location_lon != null) {
         body.addAll({'selected_location_lon': report.selected_location_lon});
       }
+      if (report.displayCity != null) {
+        body.addAll({'display_city': report.displayCity});
+      }
```

Comment:
The API does not accept this parameter... Need to be removed. In the future we will make the backend fetch the city name and expose it through the API.